plugins/chat/app/models/chat_message.rb

```diff
@@ -82,15 +82,15 @@ def attach_uploads(uploads)
     UploadReference.insert_all!(ref_record_attrs)
   end
 
-  def excerpt
+  def excerpt(max_length: 50)
     # just show the URL if the whole message is a URL, because we cannot excerpt oneboxes
     return message if UrlHelper.relaxed_parse(message).is_a?(URI)
 
     # upload-only messages are better represented as the filename
     return uploads.first.original_filename if cooked.blank? && uploads.present?
 
     # this may return blank for some complex things like quotes, that is acceptable
-    PrettyText.excerpt(cooked, 50, {})
+    PrettyText.excerpt(cooked, max_length, { text_entities: true })
```

Comment:
This `text_entities` addition makes it so `&hellip;` just becomes `...` for easier comparison in system specs